debian/control

```diff
@@ -14,7 +14,7 @@ XS-Testsuite: autopkgtest
 Package: unattended-upgrades
 Architecture: all
 Depends: ${shlibs:Depends}, ${misc:Depends}, debconf, python3, 
- python3-apt, apt-utils, apt, ucf, lsb-release, lsb-base (>= 3.2-14),
+ python3-apt, apt-utils, apt (>= 1.4.2~), ucf, lsb-release, lsb-base (>= 3.2-14),
```

Comment:
I'm proposing a patch against apt instead and u-u then can drop the versioned dependency.
Please don't merge before I update this PR.